docs/asciidoc/modules/ROOT/pages/deprecations-and-additions/index.adoc

```diff
@@ -0,0 +1,515 @@
+----
+apoc.text.doubleMetaphone(value) yield value
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.text.doubleMetaphone(text) yield value
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.math.round(value,[prec],mode=[CEILING,FLOOR,UP,DOWN,HALF_EVEN,HALF_DOWN,HALF_UP,DOWN,UNNECESSARY])
+----
+a|
+Replaced by the Neo4j round() function:
+[source, cypher, role="noheader"]
+----
+RETURN round(3.141592, 3)
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.warmup.run(loadProperties=false,loadDynamicProperties=false,loadIndexes=false)
+----
+a|
+This procedure duplicated functionality of page cache warm up which is a part of the DBMS.
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.date.expire(node,time,'time-unit')
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.ttl.expire(node,time,'time-unit')
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.date.expireIn(node,time,'time-unit')
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.ttl.expireIn(node,timeDelta,'time-unit')
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.mongodb.get()
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.mongo.find(uri, query, $config) yield value
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.mongodb.count(host-or-key,db,collection,query) yield value
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.mongo.count(uri, query, $config) yield value
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.mongodb.first() yield value
+----
+a|
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.mongodb.find() yield value
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.mongo.find(uri, query, $config) yield value
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.mongodb.insert(host-or-key,db,collection,documents)
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.mongo.insert(uri, documents, $config) yield value
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.mongodb.delete(host-or-key,db,collection,query)
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.mongo.delete(uri, query, $config)
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.mongodb.update(host-or-key,db,collection,query,update)
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.mongo.update(uri, query, update, $config)
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.cluster.graph()
+----
+a|
```

Comment:
For my own curiosity, do you know why this was deprecated?

Reply:
The commit said that it was to remove enterprise dependencies (assuming the commit reason is the same reason for the deprecation :P )